phpunit/functional/ToolboxTest.php

```diff
@@ -1637,4 +1639,52 @@ public function testComputeRedirect(string $interface, string $where, ?string $r
 
         $this->assertSame($result, $instance->computeRedirect($where));
     }
+
+
+    private function setUpLogFiles(): void
+    {
+        // remove the log file if it exists
+        assert(!file_exists($this->getCustomLogFilePath()) || unlink($this->getCustomLogFilePath()));
+        assert(!file_exists($this->getDefaultFilePath()) || unlink($this->getDefaultFilePath()));
```

Comment:
It does not seems usefull for the added tests.

```suggestion
```

Reply:
nop, it was the previous init in setUp()